media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Upstream seems to use the following version:

     VERSIONSTRING => '1.1.15.2',

Maybe you want to account for that .2 somehow? It seems to have been added in https://github.com/squentin/gmusicbrowser/commit/85ec2784365edfcf435a494119e6a4ba3791eb64.